src/util/scrollbar.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable, Inject} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class ScrollBar {
+  constructor(@Inject(DOCUMENT) private _document) {}
+
+  compensate() {
+    if (!this._isPresent()) {
+      return () => {};
```

Comment:
nit: this is always the same, you could extract it as `const noop = () => {};`

Reply:
Sure I will do